sdk/communication/communication-chat/swagger/README.md

```diff
@@ -11,7 +11,7 @@ description: Chat Client
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-input-file: https://int.chatgateway.trafficmanager.net/swagger/2020-11-01-preview3/swagger.json
+input-file: https://int.chatgateway.trafficmanager.net/swagger/2021-01-27-preview4/swagger.json
```

Comment:
Can we point to a specific git commit in the spec repo instead? A url like this is a moving target and there is no guarantee that regenerating would produce the same output because the swagger could have changed.

cc @amrelroumy-msft